vertica_python/vertica/connection.py

```diff
@@ -57,14 +57,10 @@ def __exit__(self, type, value, traceback):
     #
     # To support vertica_python 0.1.9 interface
     #
-    def query(self, query, handler=None):
```

Comment:
With iterate, passing the handler to the ctor is not needed. Having a handler embedded in the cursor also created several inconsistency.